composer.json

```diff
@@ -9,12 +9,12 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "4.2.*|5.0.*|5.1.*"
+        "illuminate/support": "4.2.*|5.0.*|5.1.*|5.2.*"
```

Comment:
True. But when 5.3 comes out you'll need to change this file.

http://semver.mwl.be/#?package=illuminate%2Fsupport&version=%3E%3D4.2&minimum-stability=stable